integrationTests/testInitializer.go

```diff
@@ -318,11 +318,14 @@ func CreateGenesisBlocks(
 	uint64Converter typeConverters.Uint64ByteSliceConverter,
 	metaDataPool dataRetriever.MetaPoolsHolder,
 	economics *economics.EconomicsData,
+	rootHash []byte,
 ) map[uint32]data.HeaderHandler {
 
 	genesisBlocks := make(map[uint32]data.HeaderHandler)
 	for shardId := uint32(0); shardId < shardCoordinator.NumberOfShards(); shardId++ {
-		genesisBlocks[shardId] = CreateSimpleGenesisBlock(shardId)
+		genesisBlock := CreateSimpleGenesisBlock(shardId)
+		//genesisBlock.ValidatorStatsRootHash = rootHash
```

Comment:
commented code, useless use of genesisBlock variable. Can revert to
```GO
genesisBlocks[shardId] = CreateSimpleGenesisBlock(shardId)
```

Reply:
done.